### ISSUE TITLE
High contrast theme - scrollbar thumb size is innacurate

### ISSUE THREAD

**Issue by bgashler1** [Microsoft/vscode#3682]
It allows you to pull the scroll thumb further down than the bottom of the scroll track.

**Comment by bgashler1** [comment]
![image](https://cloud.githubusercontent.com/assets/11839736/13503800/37f5dce8-e126-11e5-9689-6e0caa8cbdf9.png)

**Comment by bgashler1** [comment]
Found out the reason why this happening:
It's the `border` property we're using with a large width.  It's going on top and on bottom, which is making the size of the thumb too large.  What we should be using is an `outline` with an `outline-offset`.

However, from a usability perspective, I noticed that there is room to improve the way we do it while we fix this issue.  We are using very strong colors to meet AAA color accessibility requirements, but this makes it hard to see what's behind the thumb.  What if we used a background color that is less strong and used an outline that is strong, that way users can still see behind the thumb and see their code outline?

Pictured below is the current way (which is too difficult to see)
<img width="552" alt="screen shot 2016-03-03 at 10 34 12 am" src="https://cloud.githubusercontent.com/assets/11839736/13504947/9aebd01e-e12b-11e5-8241-dc56c97eb5a3.png">
<img width="551" alt="screen shot 2016-03-03 at 10 34 31 am" src="https://cloud.githubusercontent.com/assets/11839736/13504946/9aea95be-e12b-11e5-964c-de0fa7a497f7.png">
